oncall/test-infra.yaml

```diff
@@ -0,0 +1,48 @@
+args:
+- --victims=fejta,ixdy,rmmh,krzyzacy,cjwagner
+- --extend_at_least=4
+- --extend_upto=6w
+- --shift_switch_dates=Wed 12:00
+shifts:
+- Feb 01 12:00 2017 | ixdy         // no-oncall fejta(out)
```

Comment:
should this be using UTC instead?

Reply:
I'm fine with it being the median timezone (PST), but maybe we should declare that explicitly?

Reply:
yeah, this should at least be more clearly documented. I guess using UTC makes handling DST more annoying.